packages/loopring_v3/contracts/amm/LoopringAmmPool.sol

```diff
@@ -140,10 +183,29 @@ contract LoopringAmmPool is
         state.withdrawWhenOffline();
     }
 
-    function updateExchangeOwnerAndFeeBips()
+    function depositAssetsToL2(
+        uint96[] memory amounts
+        )
+        external
+        nonReentrant
+        onlyWhenOnline
```

Comment:
When the pool is offline all funds need to be returned to L1 so the LPs can withdraw on L1 (same behavior as before). So when the pool is offline we cannot allow the funds to be deposited back because than another force withdrawal would be needed to get the funds back to L1 (but the operator can then deposit to L2 again so could be a never ending loop if the operator wants to be difficult).